### PR TITLE
Update Readme to include instructions to install pods

### PR DESCRIPTION
**Pull request by cmcaboy**
# Overview

Although it seems obvious, newer developers may not know to run install pods after installing this library. This PR adds a simple line to the readme file instruction the user to run pod install.

# Test Plan

It's an update to a readme file. No testing needed.

### CODE REVIEW

**Comment by Naturalclar** [comment]
@cmcaboy Thanks for the suggestion!
Though, the README already includes instructions for running `npx pod-install` to install Pod files for iOS, which is simpler and preferred way of linking the package.

```
React Native v0.60+
For iOS, use cocoapods to link the package.

run the following command:

$ npx pod-install
```

**Comment by cmcaboy** [comment]
@Naturalclar The way the docs are laid out, it looks like npx pod-install is only applicable for react native 0.60 and above. There is nothing for 0.59 and below. pod instal is still required for 0.59 and below.

**Comment by Naturalclar** [comment]
@cmcaboy For React Native versions 0.59 and below, autolinking of native modules are not available yet, so neither npx pod-install or pod install will work. You will need to run `react-native link @react-native-community/clipboard` .

**Comment by cmcaboy** [comment]
@Naturalclar Running react-native link @react-native-community/clipboard adds the pod to your podfile, but it does not run pod install. The user has to explicitly run pod install after the link command in order to install the module. This is why I created the PR.

File: README.md [hunk 1]
```diff
@@ -50,6 +50,12 @@ run the following command to link the package:
 $ react-native link @react-native-community/clipboard
 ```
 
+For iOS, make sure you install the pod file.
+
+```
+cd ios && pod install && cd ..
```
Comment (listepo-alterpost) [inline README.md:56]: npx pod-install

**Comment by cmcaboy** [comment]
@cesargdm Thank you for approving this PR!

**Pull request merged**